screens/Dashboard.js

```diff
@@ -23,10 +26,9 @@ class Dashboard extends Component {
   render() {
     const myData = this.state.StreaksData[0]
     return (
-      <view>
```

Comment:
This looks awesome. I am wondering if you will also include the "Purple Group", "Learn React Native", "as well as the streak" to the group component? I am working on the coloring part of the streak (should be able to unblock you soon).

Reply:
I think Tony was planning on assembling these into the Group Component, but if not I can do that